### ISSUE TITLE
Only the 404 error page from TYPO3 is applied

### ISSUE THREAD

**Issue by boneyfantaseas** [TYPO3-Headless/nuxt-typo3#281]
https://github.com/TYPO3-Headless/nuxt-typo3/blob/04c6409ef2b6344957164dc2a63b337eb15ded17/lib/plugins/api.js#L23
According to this line we can only use the 404 error page from within TYPO3. Is there a reason to do so? I suggest changing this line to 

`error.response.status >= 400 &&`

in order to be able to maintain all error codes in TYPO3.

**Comment by mercs600** [comment]
@boneyfantaseas yes, it was intended only for 404 and initialData fallback, but is ok to handle more error codes there. PR is welcome. 
You can also attach your own error interceptor on $typo3.api.$http object
